Lib/random.py

```diff
@@ -120,6 +120,11 @@ def __init_subclass__(cls, /, **kwargs):
                 cls._randbelow = cls._randbelow_without_getrandbits
                 break
 
+        if cls.randbytes == _random.Random.randbytes:
```

Comment:
I would add the test for `getrandbits`. No need to override `randbytes` if `getrandbits` is not overriden.

Also, this does not work good for
```python
class R(Random):
    def getrandbits(self, n): ...
    randbytes = Random.randbytes
```

Reply:
Fixed